test/unit/full_text_search/searcher_test.rb

```diff
@@ -82,5 +82,41 @@ def test_syntax_error_query
       assert_equal([issue],
                    targets.collect(&:source_record).sort_by(&:id))
     end
+
+    def test_results_ordered_by_descending_registered_time
+      parameters = {
+        order_target: "registered_time",
+        order_type: "desc",
+        search_id: @project.id,
+        news: "1",
+        attachments: "0",
```

Comment:
I couldn't understand why the value of attachments parameter is '1' at first.
So if I don't use this parameter, the searched result includes targets related to attachments.